### PR TITLE
[SR] Quadratic - add screen reader support for Quadratic interactive graph

### PR DESCRIPTION
**Pull request by nishasy**
## Summary:
Add the aria label and descriptions for the full graph and interactive elements
in the Quadratic graph, based on the [SRUX doc](https://khanacademy.atlassian.net/wiki/spaces/LC/pages/3460169729/Quadratic).

NOTE: The SRUX doc doesn't yet cover some edge cases, so I handled those to the
best of my ability.

Issue: https://khanacademy.atlassian.net/browse/LEMS-2486

## Test plan:
`yarn jest packages/perseus/src/widgets/interactive-graphs/graphs/quadratic.test.tsx`

Storybook
- http://localhost:6006/iframe.html?globals=&args=&id=perseuseditor-widgets-interactive-graph--interactive-graph-quadratic&viewMode=story
- Use the SR tree to confirm everything seems to be okay
- Use a screen reader to make sure the experience is actually okay

### CODE REVIEW

File: packages/perseus/src/widgets/interactive-graphs/graphs/quadratic.test.ts
Comment (nishasy) [inline packages/perseus/src/widgets/interactive-graphs/graphs/quadratic.test.ts:1]: Renamed to `quadratic.test.tsx` since it has React components now.

**Review by SonicScrewdriver — APPROVED** [review]
I don't have a lot of the context for the holistic approach to screenreader strings in Interactive graph, but gosh if this isn't just a _gorgeous_ PR. 

Brilliant work, and what a great improvement for screen readers!

File: packages/perseus/src/widgets/interactive-graphs/graphs/quadratic.tsx [hunk 1]
```diff
@@ -98,3 +164,72 @@ export const getQuadraticCoefficients = (
...
     return [a, b, c];
 };
+
+function getQuadraticGraphDescription(
+    state: QuadraticGraphState,
+    i18n: I18nContextType,
+): string {
+    const strings = describeQuadraticGraph(state, i18n);
+    // This particular string will never be undefined.
+    return strings.srQuadraticInteractiveElements || "";
```
[the remark this answers is not in the document]
Comment (nishasy) [reply packages/perseus/src/widgets/interactive-graphs/graphs/quadratic.tsx:174]: Unfortunately, some of the other strings can be undefined. It's only that one string that is always defined. I can create a new type that describes all the strings individually.
